tools/fastp/fastp.xml

```diff
@@ -0,0 +1,277 @@
+<tool id="fastp" name="fastp" version="0.12.3">
+    <description>Fast all-in-one preprocessing for FastQ files</description>
+    <requirements>
+        <requirement type="package" version="0.12.3">fastp</requirement>
+    </requirements>
+    <command><![CDATA[
+
+        ln -s '$in1' input1.fq &&
+        #if $paired_end_data:
```

Comment:
This can be true, but still $in2 can be false because it can be optional.

Reply:
I tried `#if str('$in2'):` but it is always true because the default value of the input is "nothing selected"

Reply:
How do you recommend to fix it?

Reply:
conditionals :(